### ISSUE TITLE
use of the dataset test_x

### ISSUE THREAD

**Issue by yangchangli** [luisfredgs/cnn-hierarchical-network-for-document-classification#3]
Hello, I have a question to ask you. In your code, I didn't find the use of the dataset test_x. Did you not perform the final test?
My English is not very good, please let me know if there is something offensive.

**Comment by luisfredgs** [comment]
Test set was fixed on model.fit() method. This test_x var was unused, will be removed.

**Comment by yangchangli** [comment]
OK，thank you very much

**Comment by CHENhush** [comment]
> OK，thank you very much

你好请问  你复现过这篇代码吗 我跑了一下yelp 只能跑到50%不到

**Comment by CHENhush** [comment]
> OK，thank you very much
我知道那里出了问题  如果看到能回复我一下吗  谢谢
